### DISCUSSION TITLE
Full Stack JavaScript Engineer

### DISCUSSION THREAD

**Discussion by jmrk7** [idurar/idurar-erp-crm#212] [Q&A]

**Comment by suraj1s** [comment]
Hi, Idurar Team, I am learning the MERN stack and currently working on a personal E-commerce website.   I want to contribute to this Project to get some real-world exposure so how can I contribute to this project?

**Reply by inesmrad** [reply]
Hi, you can check the README.md file to get to the contribution guideline

**Reply by bruno-keiko** [reply]
hello. How can I log in to the app in localhost? Thank you!

**Comment by MSNoorul** [comment]
hi how to login to idurar in local host
